src/configuration.h

```diff
@@ -152,10 +148,16 @@ class MetadataAgentConfiguration {
   }
 
  private:
-  friend class MetadataAgentConfigurationTest;
+  friend int ::main(int ac, char** av);
 
-  void ParseConfigFile(const std::string& filename);
-  void ParseConfiguration(std::istream& input);
+  void ParseConfigurationStream(std::istream& input);
```

Comment:
This renaming of private functions now seems gratuitous. Can we rename them back to their original names?